cmake/patch-activate.sh

```diff
@@ -58,6 +58,11 @@ done
 
 cd $VIRTUAL_ENV/bin
 
+# We try two variants of the first hunk and three of the second hunk,
+# corresponding to the different versions of the activate script created
+# with mkvirtualenvwrapper and venv from the standard Python library for
+# Python 3.7, 3.9 and 3.11.
```

Comment:
Why not 3.10?

Reply:
I assumed that the activate script for 3.10 is the same as for 3.9 or 3.11.